src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -78,23 +78,23 @@
  * and sample for VCF files) and then optionally aggregates it by library, sample or file, to increase power and provide
  * results at the desired resolution. Output is in a "Moltenized" format, one row per comparison. The results will
  * be emitted into a metric file for the class {@link CrosscheckMetric}.
- * In this format the output will include the LOD score and also tumor-aware LOD score which can
- * help assess identity even in the presence of a severe loss of heterozygosity with high purity (which could
+ * In this format, the output will include both the LOD score and the tumor-aware LOD score; the latter can
```

Comment:
We may want to expand upon this documentation, I just fixed up a few typos I noticed. I also found it confusing that this text was mostly---but not faithfully---reproduced in the tool summary (and hence did not fix the corresponding typos there).

Reply:
One other comment while I'm here---"moltenize/d" is a very idiosyncratic term to describe this type of table reshaping. Why not just "molten/melted"?

Reply:
I have no strong feelings about moltenized....if you think that "molten" is more clear, go for it!